packages/dialtone-vue3/recipes/conversation_view/editor/index.js

```diff
@@ -1,2 +1 @@
-export { default as DtRecipeWysiwygEditor } from './editor.vue';
-export {} from './editor_constants.js';
+export { default as DtRecipeEditor } from './editor.vue';
```

Comment:
This was not being used AFAIK so renaming shouldn't break anything and it's in sync with Vue 2 naming.